plugins/main/public/components/common/welcome/dashboard/events-count.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { AlertsDataSource } from '../../data-source/pattern/alerts/alerts-data-source';
+import { AlertsDataSourceRepository } from '../../data-source/pattern/alerts/alerts-data-source-repository';
+import { getPlugins } from '../../../../kibana-services';
+import { getDashboardPanels } from './dashboard_panels';
+import { ViewMode } from '../../../../../../../src/plugins/embeddable/public';
+import { useDataSource } from '../../data-source/hooks';
+import { PatternDataSource, tParsedIndexPattern } from '../../data-source';
+import {
+  EuiPanel,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+import { useTimeFilter } from '../../hooks';
+import { LoadingSpinner } from '../../loading-spinner/loading-spinner';
+
+const plugins = getPlugins();
+const DashboardByRenderer = plugins.dashboard.DashboardContainerByValueRenderer;
+
+export const EventsCount = () => {
+  const {
+    dataSource,
+    fetchFilters,
+    isLoading: isDataSourceLoading,
+  } = useDataSource<tParsedIndexPattern, PatternDataSource>({
+    DataSource: AlertsDataSource,
+    repository: new AlertsDataSourceRepository(),
+  });
+
+  const { timeFilter } = useTimeFilter();
```

Comment:
**thought**: why is this using the global `timefilter` instead of using the date range coming from the 
`useDataSource`?

Reply:
The useDataSource and useSearchBar hooks will be implemented when replacing the Datepicker with the SearchBar component in a future iteration. This PR is focused on removing the agent controller.